src/Facades/StravaClient.php

```diff
@@ -11,6 +12,6 @@ class StravaClient extends Facade
 {
     protected static function getFacadeAccessor(): string
     {
-        return \JordanPartridge\StravaClient\StravaClient::class;
+        return StravaClientService::class;
```

Comment:
I think I'm a fan of this Alias format, but Maybe this Class should be actually called a service and namespaces in Services?